### PR TITLE
Cosmetic changes (2)

### PR DESCRIPTION
**Pull request by mjepronk**
- Created measureit_arch_utils.py (in order to prevent cyclic import)
- Fixed more flake8 warnings

### CODE REVIEW

File: measureit_arch_geometry.py [hunk 1]
```diff
@@ -2799,9 +2795,18 @@ def generate_end_caps(context, item, capType, capSize, pos, userOffsetVector, mi
     return capCoords, filledCoords
 
 
-def generate_text_card(context, textobj, textProps,
-                       rotation=Vector((0, 0, 0)), basePoint=Vector((0, 0, 0)),
-                       xDir=Vector((1, 0, 0)), yDir=Vector((0, 1, 0)), cardIdx=0):
+def generate_text_card(
```
Comment (mjepronk) [inline measureit_arch_geometry.py:2798]: Be careful with mutable objects as default values in keyword arguments!

**Comment by mjepronk** [comment]
I've changed the way the addon is loaded and classes are registered. We can now use `F3` -> `Reload Scripts` in Blender to reload this extension during development.

Let me know what you think about this.

Also, I removed the preset code, if you want to keep it let me know.

**Comment by kevancress** [comment]
Hey @mjepronk 

I've merged this and the papers sizes PR's to the development branch!

I'm trying to adjust the structure of my repo to be more in line with the practices outlined here : https://nvie.com/posts/a-successful-git-branching-model/ (doing most of the dev work on the development branch and keeping master for major releases milestones) so master is a bit out of date (I really need to mark a proper release and update it, but I haven't gotten around to it yet.)

I've also made the development branch the default branch for PR's going forward.

 The one thing I had some issues with was the local_attrs utility when rendering images. It kept resulting in a name error when rendering images. This is probably because I messed something up while doing the merge though. I've left the old code there for now, but if you can take a look and get the local_attrs utility working again that would be fantastic!

Thanks!!!!!
Kevan

**Comment by kevancress** [comment]
also there are some backwards compatibility issues between the development branch and the current master, dimension added with versions < 0..4.6, get erased when the file is loaded with the development branch enabled, just a heads up if your using the addon on any major projects.

**Comment by mjepronk** [comment]
> I've merged this and the papers sizes PR's to the development branch!
> 
> I'm trying to adjust the structure of my repo to be more in line with the practices outlined here : https://nvie.com/posts/a-successful-git-branching-model/ (doing most of the dev work on the development branch and keeping master for major releases milestones) so master is a bit out of date (I really need to mark a proper release and update it, but I haven't gotten around to it yet.)

OK!

> The one thing I had some issues with was the local_attrs utility when rendering images. It kept resulting in a name error when rendering images. This is probably because I messed something up while doing the merge though. I've left the old code there for now, but if you can take a look and get the local_attrs utility working again that would be fantastic!

OK, I'll create a PR for an updated `measurit_arch_render.py`.

> Thanks!!!!!
> Kevan

Thank you!
-- Matthias

**Pull request closed without merging**